src/ai/backend/manager/models/kernel.py

```diff
@@ -22,6 +22,7 @@
 import sqlalchemy as sa
 from dateutil.parser import parse as dtparse
 from dateutil.tz import tzutc
+from dateutil.tz.tz import tzfile
```

Comment:
This could be merged to the above line.  Maybe your editor got confused about nested imports. :wink:
`from dateutil.tz import tzfile, tzutc`